src/content/administration/okteto-insights.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Okteto Insights
+description: This page provides information about Okteto Insights feature.
+sidebar_label: Okteto Insights
+id: okteto-insights
+---
+
+Okteto inherently captures a lot of data about user's usage. This consists of product usage and behavior (i.e. developers) data and infrastructure utilization and performance (i.e. admins) data. But this data is not easily available today for Okteto admin users. The idea behind Okteto Insights is to provide a way for Okteto administrators to get and consume this data.
+
+By default, this feature is disabled. To enable it you just need to set the helm setting [`insights.enabled`](/self-hosted/administration/configuration.mdx/#insights) to `true` and upgrade your Okteto instance.
+
+## How to consume Okteto Insights' data
+
+Once you enable the feature, Okteto will create a new endpoint accesible through `https://okteto.SUBDOMAIN/metrics` where data will be available in Prometheus format. This new endpoint is protected with a bearer token which is auto-generated by Okteto, or you can specify your own through the secret defined under this [helm setting](/self-hosted/administration/configuration.mdx/#insights).
+
+Then, you can configure a scrapper to connect to this endpoint using the proper token. An example of a Prometheus scrapper configuration is:
+
+```yaml
+      - job_name: 'okteto_scrapper'
+        scrape_interval: 5m
+
+        scheme: http
+        static_configs:
+          - targets: ['okteto.<SUBDOMAIN>:443'] # replace <SUBDOMAIN> with your Okteto subdomain
+        authorization:
+          type: Bearer
+          credentials: <bearer-token> # replace <bearer-token> with the token generated by Okteto
+```
+
+> Be aware that Okteto does not provide a Prometheus instance or storage for this data, so you need to deploy your own Prometheus instance (or other collector platform) to consume this data and take care of counters and metrics restart with the functions Prometheus provides.
+
+## What data is available
+
+Okteto will progressively add more data exposed through this endpoint, but the following metrics are available today:
+
+- `okteto_system_node`: This metric is a [gauge](https://prometheus.io/docs/concepts/metric_types/#gauge) which represents the number of nodes running Okteto. This metric doesn't have any label.
```

Comment:
```suggestion
- `okteto_system_node`: This metric is a [gauge](https://prometheus.io/docs/concepts/metric_types/#gauge) which represents the number of nodes running Okteto. This metric does not have any label.
```

@ifbyol just to confirm: "nodes running Okteto" -- I want to double check this wasn't intended to be "nodes running _in_ Okteto" or something similar. I'm assuming you meant what's written here, but I hadn't seen this phrasing before so thought I'd ask 😄

Reply:
I wanted to say: "This is the number of nodes in the Kubernetes cluster that are used to run Okteto and all the services and namespaces created/deployed through Okteto"